js/os.ts

```diff
@@ -25,6 +25,23 @@ interface CodeInfo {
   sourceCode: string | undefined;
 }
 
+/** Get path to current deno's executable
+ *
+ *       import { execPath } from "deno";
+ *       console.log(execPath());
+ */
+export function execPath(): string {
```

Comment:
`execPath` or `executable` ?

@piscisaureus @kitsonk @kevinkassimo @kt3k @bartlomieju do you have an opinion?

Reply:
It'll be easy to rename

Reply:
@ry maybe `denoExecutable`?

Reply:
Prefer `execPath`. If named `executable` I would assume a `deno.FileInfo` returned instead of just a string for path

Reply:
Don't let me name Ali's, as I would choose `executablePath` 😁

Reply:
Given that Node uses `execPath` and Python uses `executable` I'm with @kitsonk on `executablePath`

Reply:
Ok, let's stick with `execPath` (tho I prefer `executable`)